preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -582,16 +587,33 @@ def concatenate(
 
         number_fields = len(input_data.keys()) + 1
 
-        accession_version = args["accession_version"]
+        if not isinstance(args["accession_version"], str):
```

Comment:
Ugh this is quite horribly verbose - I don't think we need to add that much code here?